ur_manipulator:
  #kinematics_solver: trac_ik_kinematics_plugin/TRAC_IKKinematicsPlugin
  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
  kinematics_solver_search_resolution: 0.0050000000000000001
  kinematics_solver_timeout: 10.0
  kinematics_solver_attempts: 5
  tip_link: ${tf_prefix}tcp
